### DISCUSSION TITLE
jscompress.com 5.0.x

### DISCUSSION THREAD

**Discussion by RadrigoFomins** [twbs/bootstrap#33979] [General] [answered]
При попытке сжать js v5.0.1 на jscompress.com показывает ошибки. Скрипты версии v4.6.0 сжимает без проблем.

https://b.radikal.ru/b20/2105/f4/43f93c954af0.png

File event-handler.js: Unexpected token: punc «{», expected: punc «;» (line: 55, col: 20)
File data.js: Unexpected token: name «doesn», expected: punc «,» (line: 36, col: 33)
File manipulator.js: Unexpected token: punc «{», expected: punc «,» (line: 39, col: 43)
File base-component.js: Unexpected token: name «BaseComponent», expected: punc «;» (line: 111, col: 8)
File modal.js: Unexpected token: punc «{», expected: punc «;» (line: 24, col: 15)
File carousel.js: Unexpected token: punc «{», expected: punc «;» (line: 24, col: 15)
File collapse.js: Unexpected token: punc «{», expected: punc «;» (line: 22, col: 15)
File tab.js: Unexpected token: punc «{», expected: punc «;» (line: 34, col: 22)
File dropdown.js: Unexpected token: punc «{», expected: punc «;» (line: 43, col: 15)

**Comment by XhmikosR** [comment, marked as answer]
You should use another tool (like terser or uglify-js) which support ES6+ syntax.
